CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## [Unreleased]
 
+-   Ability to pass extra environment variables to the Kedro nodes using `--env-var` option
```

Comment:
Let's also add `--env-var GETINDATA=ROCKS!` to e2e tests to cover this feature.

Reply:
:sunglasses: